python/invoke_release/tasks.py

```diff
@@ -1438,17 +1437,23 @@ def release(_, verbose=False, no_stash=False):
         if USE_PULL_REQUEST:
             if current_branch_name != BRANCH_MASTER:
                 _checkout_branch(verbose, current_branch_name)
+            pr_opened = ''
```

Comment:
maybe set `pr_opened = False` inside `KeyError` instead of here, seems cleaner

Reply:
I had it implemented like that and went this way to make `pr_opened` always a string. I agree that the boolean es cleaner though